AxonDeepSeg/integrity_test.py

```diff
@@ -17,48 +22,70 @@
 from config import axonmyelin_suffix
 
 def integrity_test():
+        try:
+
+            # get path of directory where AxonDeepSeg was installed
+            dir_path = Path(__file__).resolve().parent
```

Comment:
I think the reason why the GitHub diff is so weird is because `integrity_test` now has an indentation of 8 spaces?

I think if you revert this back to 4 spaces, the GitHub diff should end up looking much neater.

Reply:
Ahh woops - I had an added if/else while testing things out and didn't reset the tabs - thanks!